Harmony/Public/PatchJsonConverter.net5.cs

```diff
@@ -0,0 +1,57 @@
+using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace HarmonyLib
+{
+	internal class PatchJsonConverter : JsonConverter<Patch>
+	{
+		public override Patch Read(
```

Comment:
While many people disagree, I actually like long lines and they are used all over the project.